metadata-db-app/int_test/cmr/metadata_db/int_test/concepts/concept_save_spec.clj

```diff
@@ -45,6 +46,7 @@
   "Save a concept once then save it again with the provided revision id, validating that the
   response matches the expected response, and possibly that the second save succeeded."
   [concept exp-status new-revision-id exp-errors]
+  (time-keeper/clear-current-time!)
```

Comment:
Ideally, we should find the test that freezes the time without clearing it and fix it there. But this is OK.